### PR TITLE
Allow phone numbers of any format

### PR DESCRIPTION
**Pull request by mfmendiola**
Resolve: https://github.com/carekit-apple/CareKit/issues/39

### CODE REVIEW

File: CareKit/Connect/OCKConnectDetailViewController.m [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
  this list of conditions and the following disclaimer in the documentation and/or
  other materials provided with the distribution.
- 
+
  3.  Neither the name of the copyright holder(s) nor the names of any contributors
  may be used to endorse or promote products derived from this software without
  specific prior written permission. No license is granted to the trademarks of
  the copyright holders even if such marks are included in this software.
- 
+
```
Comment (YuanZhu-apple) [inline CareKit/Connect/OCKConnectDetailViewController.m]: Can you revert the style changes in license header? 
Prefer to keep them across all the headers.

Comment (mfmendiola) [inline CareKit/Connect/OCKConnectDetailViewController.m]: done 😄

File: CareKit/Connect/OCKConnectDetailViewController.m [hunk 2]
```diff
@@ -284,4 +287,4 @@ - (void)mailComposeController:(MFMailComposeViewController *)controller didFinis
     }
 }
 
-@end
+@end
```
Comment (YuanZhu-apple) [inline CareKit/Connect/OCKConnectDetailViewController.m]: Make a newline at file end.

Comment (mfmendiola) [inline CareKit/Connect/OCKConnectDetailViewController.m]: done!

**Comment by umerkhan-apple** [comment]
Looks good. r=m3.

@YuanZhu-apple Can you take a look as well?

**Comment by YuanZhu-apple** [comment]
r=me.

**Pull request merged**